backend/app/app_reviews.py

```diff
@@ -0,0 +1,68 @@
+from dataclasses import dataclass
```

Comment:
I do have some doubts about using ODRS in general, especially given it's not being developed at this point and doesn't provide a way to filter out snap and distribution packages.

Reply:
We already talked about this here https://github.com/flathub-infra/website/issues/50#issuecomment-1912522858

Let me know what you guys want to do in the end..